src/pages/BirthdayPicker.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-restricted-syntax */
+import React, { useState } from 'react';
+import {
+  Theme,
+  makeStyles,
+  Grid,
+} from '@material-ui/core';
+
+import DOBPicker from '../components/DOBPicker';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  home: {
+    marginTop: theme.spacing(1),
+  },
+}));
+
+type BirthdayPickerType = {
+
+};
+
+const BirthdayPicker: React.FC<BirthdayPickerType> = () => {
```

Comment:
This is an interesting situation, maybe  quirk of eslint. Basically you're setting `BirthdayPickerType` as a literal {} object. But because you've put that empty space between the opening and closing brackets, it doesn't give you a linting error violation for the `ban-types` rule.

So whether you did `type BirthdayPickerType = {}` or `typeBirthdayPickerType = object` it would give you that `ban-types` error.

A way to specify an arbitrary object in typescript could be something like type BirthdayPickerType = Record<string, unknown>` or in this case, to be even more specific, since this doesn't actually take props, would be `type BirthdayPickerType = Record<string, never>`

Yet another alternative would be to delete that type altogether and change line 21 to something like this
`const BirthdayPicker = (): JSX.Element => {`

Anyway, an interesting point of discussion, and honestly i'm not 100% sure which is considered the best approach.

Reply:
Yes, I see. I just tried `{}` and saw the `ban-types` warning. I see that giving `{}` is discouraged since it means "any non-nullish value". I must have immediately expanded the object literal into two lines and didn't notice the warning. 

`type BirthdayPickerType = Record<string, never>` makes a lot of sense and seems to be a nice way to do it.